.github/workflows/master.yml

```diff
@@ -465,7 +465,7 @@ jobs:
       - name: Run GoReleaser
         uses: goreleaser/goreleaser-action@v2
         with:
-          args: -f .goreleaser.prerelease.yml --rm-dist --skip-validate
+          args: -p 3 -f .goreleaser.prerelease.yml --rm-dist --skip-validate
```

Comment:
What does `-p 3` do? Might be worth changing to the full arg name to better self-document if that's an option.

Reply:
It's a parallel option :)